tests/nnstreamer_decoder/runTest.sh

```diff
@@ -58,6 +58,64 @@ callCompareTest testcase3.con.log testcase3.dec.log 3 "Compare for case 3" 0 0
 ## Trigger "decoder not configured"
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc ! tensor_converter ! tensor_decoder mode=direct_video ! video/mpeg ! fakesink" 4F_n 0 1 $PERFORMANCE
 
+# Check flatbuf and protobuf libraries are built
+# The tests below are valid only with flatbuf and protobuf sub-plugin
+if [[ -d $PATH_TO_PLUGIN ]]; then
+    ini_path="${PATH_TO_PLUGIN}/ext/nnstreamer/tensor_decoder"
+    if [[ -d ${ini_path} ]]; then
+        check=$(ls ${ini_path} | grep flatbuf.${SO_EXT})
+        if [[ ! $check ]]; then
+            echo "Cannot find flatbuf"
+            report
+            exit
```

Comment:
Make a SSAT test case that gives WARNING if FlatBuf does not exist.
(non-critical FAIL)
Then, skip the rest by calling report and exit.